src/group/membership/components/GroupMembershipButton.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import _ from 'lodash';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { joinGroup, leaveGroup } from 'group/groupSlice';
+import { useGroupContext } from 'group/groupContext';
+
+const GroupMembershipButton = props => {
+  const dispatch = useDispatch();
+  const { owner, groupSlug, MemberLeaveButton, MemberJoinButton } =
+    useGroupContext();
+  const {
+    data: { email: userEmail },
+  } = useSelector(state => state.account.currentUser);
+  const { fetching, group, joining } = useSelector(state =>
+    _.get(state, `group.memberships.${groupSlug}`, {})
+  );
+
+  const loading = fetching || joining;
+
+  // TODO This should just be 5 users and we should get the total count from
```

Comment:
question: was this fixed with https://github.com/techmatters/terraso-backend/pull/92? Can we get the count now?

Reply:
@paulschreiber We can, since this is a big PR, I don't think it is a good idea to add this improvement here. Can you create an issue for this improvement?